src/backend/core/migrations/0026_blob_encryption_key_id_blob_storage_location_and_more.py

```
@@ -0,0 +1,315 @@
+                    'updated_at',
+                    models.DateTimeField(
+                        auto_now=True,
+                        editable=False,
+                        help_text='date and time at which a record was last updated',
+                        verbose_name='updated on',
+                    ),
+                ),
+                (
+                    'expires_at',
+                    models.DateTimeField(
+                        default=core.models._default_upload_expiry,
```

Ruff (0.15.12)

[error] Private member accessed: `_default_upload_expiry`

(SLF001)